### ISSUE TITLE
Burn Bootloader doesn't use the correct path

### ISSUE THREAD

**Issue by Mattymerr** [GadgetFactory/DesignLab#34]
When trying to burn the bootloader, it gets most of the path wrong.
Here is the path it is trying:
"D:DesignLab-1.0.7hardwarearduinoavr/bootloaders/bootloader.file"

It is missing a lot of slashes.

**Comment by jackgassett** [comment]
Burn bootloader isn't used anymore in DesignLab. We use "Load Circuit" icon now instead.

**Comment by Mattymerr** [comment]
I would just like to add a little bit to this. This was an error that I found happened in a few places, I can't exactly remember where else it occurred. I forgot to update this when it occurred so I am very sorry about that. If I remember correctly it was partly a bug/error and partly a PBKAC as I managed to fix it after I went back to it a month later or so. Sorry for not updating this at the time, I entirely forgot about it.
